### PR TITLE
issue 53 - updated alert box

### PR DESCRIPTION
**Pull request by cepps646**

### CODE REVIEW

File: Weather.css [hunk 1]
```diff
@@ -117,4 +117,13 @@ form {
...
  }
-    
\ No newline at end of file
+
+#errorname {
+    box-sizing: border-box;
+    background:rgba(194, 62, 62, 0.747);
+    font-size: 16px;
+    font-weight: bold;
+    color: black
```
Comment (davidgrieser) [inline Weather.css:126]: Given the background color, maybe white would look better than black here?

**Review by davidgrieser — CHANGES_REQUESTED** [review]
If I cause an error but then make a successful request it doesn't clear the error.

Screen Shot 2021-08-12 at 2 17 52 PM

File: Weather.css (hunk 1 above)
Comment (DeepthiSF) [reply Weather.css:126]: sure David, we will check on this. Thanks

File: main.js [hunk 2]
```diff
@@ -7,10 +7,12 @@ var locationname = document.getElementById("locationName");
 var hightemp = document.getElementById("highTemp");
 var lowtemp = document.getElementById("lowTemp");
 var currenttemp = document.getElementById("curTemp");
+var errorMessage = document.getElementById('errorname');
 
 btn.addEventListener('submit', function (event) {
   event.preventDefault();
   var location = document.getElementById('location').value;  
+  errorMessage.innerText = '';
```
Comment (davidgrieser) [inline main.js:15]: Nice Reset

**Review by davidgrieser — APPROVED** [review]
LGTM (Look Good To Merge)

**Comment by davidgrieser** [comment]
Resolves #53

**Pull request merged**